### DISCUSSION TITLE
part-ii/2018-68/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#15] [Question comments]
Part II 2018 Galois Theory: Paper 3, Section II, I

https://questions.tripos.org/part-ii/2018-68/

**Comment by artie2000** [comment]
For (i), am I supposed to try write $x=F(\alpha)\in L^G$ as a symmetric polynomial in $\{\sigma \alpha\mid\sigma\in G\}$?

I have $F(\sigma\alpha)=F(\alpha)$, but I can't (e.g.) sum up the conjugates and divide by $|G|$, since we don't have char. 0.

**Reply by artie2000** [reply]
This is a silly approach.
